gradle-examples/composite-build/build.gradle

```diff
@@ -0,0 +1,79 @@
+buildscript {
+    String thorntailVersion = System.getProperty('thorntailVersion') ?: VERSION_THORNTAIL
+
+    repositories {
+        mavenLocal()
+        mavenCentral()
+        maven {
+            url "https://oss.sonatype.org/content/repositories/snapshots/"
+        }
+    }
+
+    dependencies {
+        classpath "io.thorntail:thorntail-gradle-plugin:$thorntailVersion"
+    }
+}
+
+repositories {
+    mavenLocal()
+    mavenCentral()
+    maven {
+        url "https://oss.sonatype.org/content/repositories/snapshots/"
+    }
+}
+
+String thorntailVersion = System.getProperty('thorntailVersion') ?: VERSION_THORNTAIL
+
+apply plugin: 'war'
+apply plugin: 'thorntail'
+
+dependencies {
+
+    compileOnly 'javax:javaee-api:8.0'
+
+    // Include the "platform project" that we have dependency on.
+    implementation platform('io.thorntail.examples.gradle:thorntail-multi-module-app:0.0.0')
+
+    // Pull in the Thorntail BOM that we are supposed to use (as dictated by our "platform-project").
+    implementation platform('io.thorntail:bom-all:')
+
+    // If the dependent project is available (as an included build), then the below version number is ignored.
+    // This is useful when you wish to test changes across multiple projects at the same time.
+    implementation 'io.thorntail.examples.gradle:library-module:0.0.0'
```

Comment:
As the comment states, the version number will be ignored if the dependency is pulled in as a composite-build project.